ajax/libs/colors/package.json

```diff
@@ -19,5 +22,15 @@
     "name": "@mrmrs",
     "email": "hi@mrmrs.cc",
     "url": "http://mrmrs.cc"
-  }
+  },
+  "npmName": "colors.css",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "css/*",
+        "js/*"
```

Comment:
For now, results are identical, but if there are unrelated files put into `css` and `js` directory in the future, we don't want to fetch them.